pep-0681.rst

```diff
@@ -203,7 +203,7 @@ customization of default behaviors:
       eq_default: bool = True,
       order_default: bool = False,
       kw_only_default: bool = False,
-      field_descriptors: Tuple[type, ...] = (()),
+      field_descriptors: Tuple[Union[type, Callable[..., Any]], ...] = (()),
```

Comment:
```suggestion
      field_descriptors: tuple[type | Callable[..., Any], ...] = (),
```
Let's use modern syntax.

Also `()` is enough to specify an empty tuple.

Reply:
We can't use that form in typing-extensions because it needs to support older versions, but the PEP targets 3.11 so we should use annotations that would be idiomatic in 3.11.